packages/jellyfish-api-core/__tests__/category/loan/closeVault.test.ts

```diff
@@ -43,11 +47,58 @@ describe('Loan', () => {
     ]
     oracleId = await tGroup.get(0).rpc.oracle.appointOracle(addr, priceFeeds, { weightage: 1 })
     await tGroup.get(0).generate(1)
-    await tGroup.get(0).rpc.oracle.setOracleData(oracleId, Math.floor(new Date().getTime() / 1000), { prices: [{ tokenAmount: '1@DFI', currency: 'USD' }] })
+    await tGroup.get(0).rpc.oracle.setOracleData(oracleId, Math.floor(new Date().getTime() / 1000), { prices: [{ tokenAmount: '1000000@DFI', currency: 'USD' }] }) // inflate DFI so that we can mint enough TSLA, we will then reset the price later
```

Comment:
`1000000@DFI` price (i hope it will) its too far against our real life :sweat_smile: 
once you gen(1000000) DFI, you can deposit it into a "master" vault then take TSLA loan to compatible with current test

Reply:
use node 0 or 1. it comes with `100000000` DFI balance for regtest.
https://github.com/DeFiCh/ain/blob/5dd54ad271d7cf1f5da1e97ee5eeec2ff08587df/src/chainparams.cpp#L825